controllers/Alerts.js

```diff
@@ -80,4 +83,27 @@ AlertRouter.put('/edit', function(req, res) {
   });
 });
 
+AlertRouter.post('/text', function(req, res) {
+  const alertToTextId = req.body.id;
+
+  Alert.findById(alertToTextId, function(err, alert) {
+    if (err) return res.status(400).send(err);
+
+    const {name, description} = alert;
+    const date = new Date().getDay();
+
+    User.find({'day': date}, function(err, people) {
```

Comment:
`const date = new Date().getDay();` numbers dates from 0-6 from sunday - sat, so it matches perfectly with our current numbering system!